modules/fedimint-lnv2-tests/tests/tests.rs

```diff
@@ -161,158 +136,3 @@ async fn refund_unpayable_invoice() -> anyhow::Result<()> {
 
     Ok(())
 }
-
-#[tokio::test(flavor = "multi_thread")]
-async fn can_make_self_payment_exactly_once() -> anyhow::Result<()> {
-    let fixtures = fixtures();
-    let fed = fixtures.new_default_fed().await;
-    let gateway_test = gateway(&fixtures, &fed).await;
-    let gateway_api = gateway_test.gateway.versioned_api().clone();
-
-    let client = fed.new_client().await;
-
-    let (invoice, receive_op) = client
-        .get_first_module::<LightningClientModule>()
-        .receive(gateway_api.clone(), Amount::from_sats(100))
-        .await?;
-
-    let (print_op, print_outpoint) = client
-        .get_first_module::<DummyClientModule>()
-        .print_money(Amount::from_sats(10000))
-        .await?;
-
-    client
-        .await_primary_module_output(print_op, print_outpoint)
-        .await?;
-
-    // this payment will fail and be refunded since the gateway does not have any
-    // liquidity in the federation yet
-    let send_op_refunded = client
-        .get_first_module::<LightningClientModule>()
-        .send(gateway_api.clone(), invoice.clone())
-        .await?;
-
-    let mut sub = client
-        .get_first_module::<LightningClientModule>()
-        .subscribe_send(send_op_refunded)
-        .await?
-        .into_stream();
-
-    assert_eq!(sub.ok().await?, SendState::Funding);
-    assert_eq!(sub.ok().await?, SendState::Funded);
-    assert_eq!(sub.ok().await?, SendState::Refunding);
-    assert_eq!(sub.ok().await?, SendState::Refunded);
```

Comment:
Do we need to delete these? These aren't verifying the same things as devimint tests. Devimint tests only verify E2E functionality, this actually verifies each state transition.

Reply:
Chatted offline, other tests cover these transitions already